.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,28 @@
+name: Node CI
+
+on: [push]
```

Comment:
whats this do and should this PR be closed because of your other branch?

Reply:
This runs unit tests on each PR, and no.